apps/backend/src/users/users.service.ts

```diff
@@ -41,4 +35,37 @@ export class UsersService {
 
     return users;
   }
+
+  async findOne(userId: number) {
+    const user = await this.usersRepository.findOneBy({ id: userId });
+
+    if (!user) {
+      throw new BadRequestException('User not found');
```

Comment:
⛏️ Perhaps it would be good to give more explicit feedback to the caller of this endpoint by providing them with the ID of the user that cannot be found (i.e., `throw new BadRequestException(User ${userId} not found);`)